tools/hsettp/hsettp.c

```diff
@@ -594,6 +606,20 @@ print_operations(FILE * const output)
                     ((uintptr_t)args.operations[i].operation_id ^
                         (uintptr_t)args.operations[j].operation_id);
 
+                args.operations[i].alias =
+                    (const char
+                         *)((uintptr_t)args.operations[i].alias ^ (uintptr_t)args.operations[j].alias);
+                args.operations[j].alias =
+                    (const char
+                         *)((uintptr_t)args.operations[i].alias ^ (uintptr_t)args.operations[j].alias);
+                args.operations[i].alias =
+                    (const char
+                         *)((uintptr_t)args.operations[i].alias ^ (uintptr_t)args.operations[j].alias);
```

Comment:
is this from clang-format ?

Reply:
Yes, it looks horrible. I wonder if something about the penalties is screwing it up.